src/setup/defaultNodes.js

```diff
@@ -45,32 +44,42 @@ const nodeTypes = [
   pool,
   poolLane,
   boundaryEvent,
-  boundaryTimerEvent,
```

Comment:
Won't this need to be put back when boundary timer is used? This removes it from the menu.